E2_ESRGAN/main.py

```diff
@@ -1,10 +1,10 @@
 import os
 import argparse
 import logging
-from lib import settings, phase_1, model
+from lib import settings, phase_1, phase_2, model
 import tensorflow as tf
 
-
+#TODO (@captain-pool): Merge phase_1 and phase_2 in one file.
```

Comment:
nit: `# TODO(@captain-pool)`. Also, move it just above the phase_1 & phase_2 calls.

Reply:
Cool :)